apps/settings/js/panels/about/update_check.js

```diff
@@ -150,7 +154,8 @@ define(function(require) {
      */
     _checkForUpdates: function uc__checkForUpdates() {
       if (!navigator.onLine) {
-        alert(this._('no-network-when-update'));
+        navigator.mozL10n.formatValue('no-network-when-update').then(
+          msg => alert(msg));
```

Comment:
so, trying to see what will happen if formatValue takes time, I delayed it here for 5s.
The result was that the UI was a bit confusing since nothing has happened.

I can add `disabled` state on the button and remove it when formatValue resolves. It could be also a good idea to add it for when the update is being performed and remove when the checking is complete.

Would that work for you @evelynhung ?